style.css

```diff
@@ -0,0 +1,390 @@
+    width: 100%;
+    display: flex;
+  }
+  .left-section {
+    text-align: left;
+    max-width: 130px;
+    width: 100%;
+    margin: 80px 20px 0 0;
+  }
+
+  .main-title {
+    display: block;
+    font-family: NexaBold;
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 34px;
+    letter-spacing: normal;
+    color: #c6cdd9;
+    position: absolute;
+  }
+
+  .banner-image {
+    max-width: 100%;
+    max-width: 400px;
+  }
+
+  .title {
+    display: none;
+  }
+
+  .right-section {
+    width: 50%;
+    text-align: left;
+  }
+
+  .description {
+    font-family: NexaLight;
+    font-size: 12px;
+    line-height: 20px;
+    margin: 38px 15px 15px 0;
+  }
+  .learnmore-link-button {
+    padding: 9px 15px;
+    font-size: 12px;
+  }
+  .container-banner {
+    margin: 0 0 0 15px;
+  }
+}
+
+.roadmap-section {
+  position: relative;
+  padding: 120px 0 50px;
+}
+
+.backgroundImg {
+  position: absolute;
+  background-image: url(https://www.chainpals.io/assets/images/guest/svgs/roadmap.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  max-width: 390px;
+  width: 100%;
+  height: 390px;
+  left: 0;
+  top: 80px;
+  z-index: -1;
+}
+.dates {
+  display: none;
+}
+
+.container-fluid {
+  max-width: 1000px;
+  padding-right: 0px;
+  padding-left: 0px;
+}
+
+.heading {
+  font-family: NexaBold;
+  text-align: center;
+  font-size: 48px;
+  font-weight: normal;
+  line-height: 56px;
+  letter-spacing: -0.05em;
+  color: #c6cdd9;
+}
+
+.roadmap-wrapper {
+  margin-top: 40px;
+}
+
+.swiper-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.roadmap-swiper {
+  max-width: 220px;
+  width: 100%;
+  font-size: 18px;
+  position: relative;
+  text-align: center;
+}
+
+.timestamp {
+  width: 100%;
+  margin-bottom: 20px;
+  color: white;
+  position: relative;
+}
+
+.status-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
+.status-wrapper::before {
+  z-index: 1;
+  content: '';
+  position: absolute;
+  width: 120%;
+  height: 23px;
+  background-color: blue;
+  top: -10px;
+  margin: 0 auto;
+  border-radius: 23px;
+}
+
+.status::before {
+  content: '';
+  position: absolute;
+  width: 16px;
+  height: 16px;
+  background-color: white;
+  border-radius: 34px;
+  top: -38px;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  z-index: 1;
+}
+
+.status {
+  background: #191c26;
+  border-radius: 14px;
+  margin: 25px 10px;
+  padding: 10px;
+  font-family: NexaLight;
+  text-align: left;
+  font-size: 18px;
+  font-weight: 300;
+  line-height: 30px;
+  letter-spacing: normal;
+  color: grey;
+  position: relative;
+  border: 6px solid #191c26;
+}
+
+.date {
+  font-family: NexaBold;
+}
+
+.roadmap-description {
+  font-size: 16px;
+  line-height: 28px;
+  margin: 0 0 20px;
+  font-weight: 300;
+  font-family: NexaLight;
+  color: #acb0b9;
+}
+
+#hides {
+  display: none;
+}
+
+@media (min-width: 724px) and (max-width: 724px) {
```

Comment:
is there any basic breakpoints for responsive @aeshapatel025

Reply:
also whats the min of same css in two way @aeshapatel025

Reply:
changes done